plugins/ScheduledReports/templates/_listReports.twig

```diff
@@ -75,12 +75,13 @@
             </td>
             <td>
                 {# download link #}
-                <a href="{{ linkTo({'module':'API', 'segment': null, 'token_auth':token_auth,
+                <a href="#"
```

Comment:
@katebutler random thought to prevent possible XSS etc... we could have for each report a hidden form like `<form action="index.php?module=API&..."><input name=token_auth type=hidden value="..." style="display:none"></form>` and then simply submit the hidden form when someone clicks on the link? see https://stackoverflow.com/questions/18406732/using-jquery-to-submit-form-from-link-outside-of-form

This way Twig can do all of the complicated escaping and we can still use `action="{{linkTo(...)}}`

Reply:
re the `href="#"`... I'm not sure if it's still an issue (not tested), but I think it may cause a scroll to the top or so under circumstances (maybe always, or when before a different anchor was active).

I think we usually use `href="javascript:void(0)"` for this reason.

Reply:
Changed